Libraries/Components/Touchable/TouchableBounce.js

```diff
@@ -44,7 +44,19 @@ const TouchableBounce = createReactClass({
      * all the touchable elements are accessible.
      */
     accessible: PropTypes.bool,
-
+    /**
+     * When `accessible` is true (which is the default) this may be called when
+     * the OS-specific concept of "focus" occurs. Some platforms may not provide
+     * focus support have the concept of focus.
+     */
+      onFocus: PropTypes.func,
```

Comment:
>   [](start = 4, length = 2)

looks like whitespace is a bit sloppy here.

Reply:
good catch!